### PR TITLE
🏷️ Expose WebCryptoService to importers

### PR DESCRIPTION
**Pull request by dmihalcik-virtru**

### CODE REVIEW

**Comment by dmihalcik-virtru** [comment]
Explicit is better

File: lib/tdf3/index.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import {
   type EncryptResult,
   type PemKeyPair,
 } from './src/crypto/declarations.js';
+export * as WebCryptoService from './src/crypto/index.js';
```
Comment (biscoe916) [inline lib/tdf3/index.ts:17]: Nit, but I'd prefer all the exports being on the bottom of the file. 

I can make the change.

Comment (dmihalcik-virtru) [reply lib/tdf3/index.ts:17]: done

**Review by danielRicaud — APPROVED** [review]

**Pull request closed without merging**